src/store/User/userController.ts

```diff
@@ -82,21 +84,24 @@ const createKdmNode = async (user: User, onMessageReceived: any) => {
   }
 }
 
-const processKadMsg = (onMessageReceived: any) => async (kadMsgObj: any) => {
+const processKadMsg = (onMessageReceived: any) => (kadMsgObj: any) => {
```

Comment:
This is not important but can we hide/wrap this reference to Kademlia protocol? e.g.:
```js
const processMessage = (onMessageReceived: any) => (msgObj: any) => {
```

Reply:
can do